### ISSUE TITLE
Dictionary Checklist

### ISSUE THREAD

**Issue by SethClydesdale** [SethClydesdale/genki-study-resources#8]
Now that the dictionary is live, the only thing that's left to do is fill it in! Since I don't want to completely halt my development of other features, I'll be updating the dictionary's definitions over time. The progress of the dictionary will be listed below.

- [x] あ
- [x] い
- [x] う
- [x] え
- [x] お
- [x] か
- [x] き
- [x] く
- [x] け
- [x] こ
- [x] さ
- [x] し
- [x] す
- [x] せ
- [x] そ
- [x] た
- [x] ち
- [x] つ
- [x] て
- [x] と
- [x] な
- [x] に
- [x] ぬ
- [x] ね
- [x] の
- [x] は
- [x] ひ
- [x] ふ
- [x] へ
- [x] ほ
- [x] ま
- [x] み
- [x] む
- [x] め
- [x] も
- [x] や
- [x] ゆ
- [x] よ
- [x] ら
- [x] り
- [x] る
- [x] れ
- [x] ろ
- [x] わ

I did a quick estimate and it looks like about 1500 words will need to be filled in. -300 already done, so about 1200 left. If a section a day is done, it should be done within about a month.

**Comment by SethClydesdale** [comment]
While adding words, I noticed some words learned in the kanji lists via compounds are NOT present in the dictionary portion of Genki. Since I want to add all the words that are learned in the book, I'm going to go through the kanji lists for lessons 3-23 and add words that were not present in the dictionary, once I've finished the above tasks. Most likely will open up a new issue for it.

Anyway, since they don't have a label for it in the legend. I'm probably going to go with 読L{#}-漢字表 to mark words that are learned in the kanji lists via compounds. 漢字表（かんじひょう） is how they referred to the kanji lists in the introduction and means as you probably guessed it: kanji "list" or "table", however, it's a list in this instance.
